core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -333,16 +342,19 @@ pub async fn withdraw<S: EthereumSigner + Clone>(
         None => address,
     };
 
-    let withdraw_handle = wallet
+    let withdraw_builder = wallet
         .start_withdraw()
         .token(ZKSYNC_TOKEN_NAME)
         .map_err(GenericError::new)?
-        .amount(withdraw_amount)
-        .to(recipient_address)
-        .send()
-        .await
-        .map_err(GenericError::new)?;
+        .amount(withdraw_amount.clone())
+        .to(recipient_address);
+    log::debug!(
+        "Withdrawal raw data. token={}, amount={}, to={}",
+        ZKSYNC_TOKEN_NAME,
```

Comment:
you have tGLM hardcoded above in `info!`

Reply:
in an upcoming PR i'm cleaning all the GNT names, will fix this there. ( to master )